proposal.rst

```diff
@@ -271,7 +326,7 @@ Use of Existing Code
 **If you already know about existing code you plan to incorporate or libraries
 you plan to use, please give details.**
 
-FILLME
+This project requires the use of existing code i.e. to import various open source file reading libraries such as Poppler, DjVuLibre and other available libraries from the Document Liberation Project.
```

Comment:
How will omega be affected by the licensing of these libraries? What licenses are they under?

Reply:
Details have been added